region/crateria/central/Crateria Super Room.json

```diff
@@ -1140,6 +1141,19 @@
       ],
       "devNote": "It's not really insane jump difficult, but that is where you think about needing to avoid the damage.  And failing the jump is still very punishing due to falling into the acid."
     },
+    {
+      "link": [5, 3],
+      "name": "Use Flash Suit",
+      "requires": [
+        {"or": [
+          "canConsecutiveWalljump",
+          "h_canFly"
```

Comment:
You can't IBJ into using a flash suit

Reply:
Added a `canUnmorphBombBoost` requirement now, since that's the "trick" to make this work: I press up to unmorph while starting to descend toward the last bomb; I keep holding up the whole time after that, which buffers the spark to activate when the last bomb boost is finished. I was just doing single bomb jumping and for me this is just what happened anyway, without specifically going for it; so there's no precise timing to it or anything. But yeah, if you don't get a bomb boost after unmorphing then it won't let you activate the spark.

Reply:
I didn't know it worked like that, but it makes sense.